src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,396 @@
+        updatePotAndStackAmount(gameState.smallBlind, playerSmallBlind);
+        System.out.println("Zaczyna " + playerSmallBlind.getNamePlayer());
+        System.out.println(playerBigBlind.getNamePlayer() + " bigBlind " + gameState.bigBlind + "$");
+        System.out.println(playerSmallBlind.getNamePlayer() + " smallBlind " + gameState.smallBlind + "$");
+        printPot();
+        System.out.println(playerSmallBlind.getNamePlayer() + " podejmuje decyzje");
+        System.out.println(playerSmallBlind.getNamePlayer() + " " + amountToCall(playerBigBlind, playerSmallBlind) + "$ to call");
+        // System.out.println("1 = call 2 = raise");
+    }
+
+    public void printPot() {
+        System.out.println("Pot wynosi " + gameState.getPot() + "$");
+    }
+
+    public void layCards(GameState gameState) {
+        boolean isDraw = false;
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            printPot();
+            System.out.println("Flop");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.FLOP) {
+            printPot();
+            System.out.println("Turn");
+            printBoard(gameState.getGamePhase());
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            printPot();
+            System.out.println("River");
+            printBoard(gameState.getGamePhase());
+        } else {
+            GameResult.result whoWins = equityEvaluator.updatePlayerWins(result.getBoard(), result.getHand1(), result.getHand2());
+            System.out.print(whoWins + " " + gameState.getPot() + "$");
+            if (whoWins == GameResult.result.DRAW) {
+                isDraw = true;
+            }
+            if (whoWins == GameResult.result.WIN_PLAYER1) {
+                updatePlayersOnShowdownResult(player1, player2, isDraw);
+            } else {
+                updatePlayersOnShowdownResult(player2, player1, isDraw);
+            }
+        }
+    }
+
+    public void printBoard(GamePhase gamePhase) {
+        int endIndex = BOARD_PRINT_INDEXES.get(gamePhase);
+        for (int i = 0; i < endIndex; i++) {
+            System.out.print(result.getBoard().get(i) + " ");
+        }
+        System.out.println();
+    }
+
+    public void makeRaisePlayer(Player activePlayer, Player nonActivePlayer) {
+        if (gameState.getGamePhase() == GamePhase.PREFLOP && nonActivePlayer.getPreviousRaise() == 0 && !limp) {
+            System.out.println("Podaj do ilu raise minimum 2$");
+        } else if (limp && nonActivePlayer.getPreviousRaise() == 0 && amountBetPlayer == 0) {
+            System.out.println("Podaj do ilu raise");
+        } else if (nonActivePlayer.getPreviousRaise() == 0) {
+            System.out.println("Podaj do ilu raise, minimum " + (amountBetPlayer * 2));
+        } else {
+            System.out.println("Podaj do ilu, Raise minimum " + (nonActivePlayer.getPreviousRaise() * 2));
+        }
```

Comment:
dobra przemyslalem sprawe i 4 ify na to to jest gowno. Tak naprawde to sie sprowadza do tego ze jak jest raise to znaczy ze wczesniej byl wprowadzony bet (limpa tez mozna potraktowac jako bet 1 (czyli ta zmienne globalne limp i firstDecisionSmallBlind sa do wyjebania)), i jakbys stworzyl sobie zmienna globalna lastBet/lastBetAmount to dzieki niej bedziesz wiedzial jaka kwota jest minimalnym raisem (lastBet * 2). Moze tez ci sie przydac zmienna w klasie Player np moneyPuttedOnStreet/moneyOnStreet (nie jestem pewny czy ci sie przyda ale mozna by tym sledzic ile kasy gracz wrzucil np na flopie i jak idziesz na turna to to zerujesz i od nowa sledzisz)